x-pack/plugins/security_solution/public/resolver/view/panel.test.tsx

```diff
@@ -160,23 +189,39 @@ describe(`Resolver: when analyzing a tree with no ancestors and two children and
 
   it('should have 3 nodes (with icons) in the node list', async () => {
     await expect(
-      simulator().map(() => simulator().testSubject('resolver:node-list:node-link:title').length)
-    ).toYieldEqualTo(3);
-    await expect(
-      simulator().map(() => simulator().testSubject('resolver:node-list:node-link:icon').length)
-    ).toYieldEqualTo(3);
+      simulator().map(() => {
+        return {
+          titleCount: simulator().testSubject('resolver:node-list:node-link:title').length,
+          iconCount: simulator().testSubject('resolver:node-list:node-link:icon').length,
+        };
+      })
+    ).toYieldEqualTo({ titleCount: 3, iconCount: 3 });
   });
 
-  it('should be able to copy the timestamps for all 3 nodes', async () => {
-    const copyableFields = await simulator().resolve('resolver:panel:copyable-field');
+  describe('when the user hovers over the timestamp for "c.ext" with their mouse', () => {
+    beforeEach(async () => {
```

Comment:
This test is a lot like the other one, but we have data in tables instead of `dl`s. I'll look for a way to improve this code.